### PR TITLE
Fixed stripping diacritics from IT fiscalization DTOs

### PR DESCRIPTION
**Pull request by abdallahbeshi**
## Description

Fixed stripping diacritics from IT fiscalization DTOs.

## Type of change
- [x] Bug fix.
- [ ] Feature.
- [ ] Consolidation.

## Checklist
- [ ] Is breaking change.
- [ ] Documentation updated.
- [ ] Tests included (please specify cases).

### CODE REVIEW

**Review by tonyjoanes — APPROVED** [review]

File: src/Italy/Mews.Fiscalizations.Italy/Dto/DtoUtils.cs [hunk 1]
```diff
@@ -26,33 +23,20 @@ public static string NormalizeZip(this string zip)
...
-            return String.IsNullOrEmpty(str) ? null : str;
-        }
-
-        private static char GetCharacterSubstitute(char character, char substituteCharacter, int highestValidCharacter)
-        {
-            return (int)character <= highestValidCharacter ? character : substituteCharacter;
-        }
+            if (string.IsNullOrWhiteSpace(s))
+            {
+                return s.NonEmptyValueOrNull();
```
Comment (KaliCZ) [inline src/Italy/Mews.Fiscalizations.Italy/Dto/DtoUtils.cs:35]: Why would we replace empty string with a null?

**Review by KaliCZ — COMMENTED** [review]
So just to confirm, you're sure that we no longer care about the max ascii character?

**Comment by abdallahbeshi** [comment]
> So just to confirm, you're sure that we no longer care about the max ascii character?

@KaliCZ The technical docs doesn't say anything about that

File: src/Italy/Mews.Fiscalizations.Italy/Dto/DtoUtils.cs (hunk 1 above)
Comment (abdallahbeshi) [reply src/Italy/Mews.Fiscalizations.Italy/Dto/DtoUtils.cs:35]: Yea that looks fishy, i just left as is for now, but will have a ticket to check the tech documentation and see if its needed or not..

**Pull request merged**